diff --git a/keyboards/cyboard/imprint/imprint_number_row_5key_bottom_row/rules.mk b/keyboards/cyboard/imprint/imprint_number_row_5key_bottom_row/rules.mk
index 8b13789179..c78129ee33 100644
--- a/keyboards/cyboard/imprint/imprint_number_row_5key_bottom_row/rules.mk
+++ b/keyboards/cyboard/imprint/imprint_number_row_5key_bottom_row/rules.mk
@@ -1 +1,3 @@
-
+CAPS_WORD_ENABLE = yes
+UNICODEMAP_ENABLE = yes
+RAW_ENABLE = yes
